content/learn/contribute/helping-out/writing-docs.md

```diff
@@ -209,28 +209,32 @@ When writing and reviewing learning material for the Bevy Book and Quick Start G
    3. Lack of polish should serve an end: don't show bad or sloppy practices without a good reason.
    4. Showing how (and why!) to refactor your code is a very powerful teaching tool.
 6. Stick to a consistent style (e.g. for loops vs map) within each example.
-7. If you need to give advice that will only matter to some of your audience (e.g. how to handle an edge case, or support a specific platform), do so in a clearly marked aside or list.
+7. If you need to give advice that will only matter to some of your audience (e.g. how to handle an edge case, or support a specific platform), do so in a clearly marked aside (like a callout), or list.
```

Comment:
```suggestion
7. If you need to give advice that will only matter to some of your audience (e.g. how to handle an edge case, or support a specific platform), do so in a clearly marked aside (like a callout) or list.
```

I don't think there should be a comma here. "aside" and "list" should be grouped together since there are only two of them, not kept apart.